com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -607,15 +630,15 @@ private SceneEventProgress ValidateSceneEvent(string sceneName, bool isUnloading
             }
 
             // Return invalid scene name status if the scene name is invalid
-            if (!IsSceneNameValid(sceneName))
+            if (SceneUtility.GetBuildIndexByScenePath(sceneName) == -1)
```

Comment:
Can we get a named constant instead of `-1` ? `InvalidIndex` or `NotFound` or something ?

Reply:
Yes! We can... good call out.